backend/app/tests/utils/test_db.py

```
@@ -313,7 +313,7 @@ def test_database_connection() -> bool:
     elif importlib.util.find_spec("psycopg2"):
         driver_name = "postgresql+psycopg2"
     else:
-        return False  # 如果两个驱动都不可用，则连接失败
+        assert False, "无法找到数据库驱动程序（psycopg 或 psycopg2）"
```

Ruff (0.11.9)

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)